docs/user-guide/patterns/sums.rst

```diff
@@ -0,0 +1,69 @@
+.. index::
+   single: sums
+
+Tracking Sums
+=============
+
+This section deals with tracking sums of values. The quintessential example for
+needing to track sums is from ERC-20 contracts, where :solidity:`totalSupply()`
+must be the sum of all balances. This is the example we shall use here,
+namely the :clink:`ERC20</DEFI/ERC20/contracts/ERC20.sol>` contract from the
+:clink:`Examples</>` repository.
+
+
+Trying to verify the sum of two balances
+----------------------------------------
+Often one needs a invariant like :cvl:`sumOfTwo` below:
```

Comment:
"a invariant"? I think it should be "an" or maybe even "the' because we talk about specific invariant. But definitely not "a"

I think the transition from the first paragraph to this part isn't clear. We need to check the sum of ALL balances, but now we downgraded our property to the sum of only two balances.
Maybe something like: "First of all, let's start with a simpler property: sum of two balances is less than or equal to `totalSupply()`. It will also improve our understanding of how the prover works and finds counter-examples."